packages/editor/src/pages/Editor/store/misc/sagas.ts

```diff
@@ -94,8 +95,28 @@ function* onConfirmSwitchEnvironmentSaga(
 }
 
 function* onPopOutEditorSaga() {
-  Office.context.ui.displayDialogAsync(window.location.href);
-  window.location.href = currentRunnerUrl;
+  Office.context.ui.displayDialogAsync(
+    window.location.href,
+    { height: 60, width: 60, promptBeforeOpen: false },
+    (result: Office.AsyncResult<any>) => {
+      if (result.status === Office.AsyncResultStatus.Succeeded) {
+        window.location.href = currentRunnerUrl;
+      } else {
+        console.error(result);
+        actions.messageBar.show({
```

Comment:
that's not going to do what you think it's going to do. you called the action creator meaning that this .show() evaluates to a json object that needs to get dispatched from the store. the way of dispatching the action to the store from a saga is yield put, but youre doing this here inside of a callback which is a function not a generator.... so this might be difficult..